NEWS.md

```diff
@@ -1,203 +1,117 @@
-3.36.1
+3.37.1
 ======
 
 ## Highlights
-
- * `system()` source: added basic support for reading macOS system logs
-
-   The current implementation processes the output of the original macOS syslogd:
-   `/var/log/system.log`.
-   ([#3710](https://github.com/syslog-ng/syslog-ng/pull/3710))
- * `$(values)` and `$(names)`: these new template functions can be used to
-   query a list of name-value pairs in the current message. The list of name
-   value pairs queried are specified by a value-pairs expression, just like
-   with `$(format-json)`.
-
-   Examples:
-
-     This expression sets the JSON array `values` to contain the list of SDATA
-     values, while the JSON array `names` would contain the associated names, in
-     the same order.
-
-     `$(format-json values=list($(values .SDATA.*)) names=list($(names .SDATA.*)))`
-
-   The resulting name-value pairs are always sorted by their key, regardless of
-   the argument order.
-   ([#3911](https://github.com/syslog-ng/syslog-ng/pull/3911))
- * `rename()`: added a new rewrite rule, called `rename()`
-
-   Example usage:
-   ```
-   rewrite {
-     rename( "renamed-from" "renamed-to" );
-   };
-   ```
-   ([#3841](https://github.com/syslog-ng/syslog-ng/pull/3841))
+  * `network()`, `syslog()` sources and destinations: added new TLS options `sigalgs()` and `client-sigalgs()`
```

Comment:
I would not list this as a highlight, it's a very minor restriction possibility in TLS.